app/MomMock/Method/Postsales/ReturnManagement/Authorize.php

```diff
@@ -0,0 +1,138 @@
+<?php
+/**
+ * Copyright (c) 2018 Magenerds
+ * All rights reserved
+ *
+ * This product includes proprietary software developed at Magenerds, Germany
+ * For more information see http://www.magenerds.com/
+ *
+ * To obtain a valid license for using this software please contact us at
+ * info@magenerds.com
+ */
+namespace MomMock\Method\Postsales\ReturnManagement;
+
+use MomMock\Entity\Order;
+use MomMock\Entity\Order\Item as OrderItem;
+use MomMock\Method\AbstractIncomingMethod;
+use MomMock\Entity\Rma;
+use MomMock\Entity\Rma\Item;
+
+/**
+ * Class Authorize
+ * @package MomMock\Method\Sales\OrderManagement
+ * @author  Harald Deiser <h.deiser@techdivision.com>
+ */
+class Authorize extends AbstractIncomingMethod
+{
+    const CREDIT_NOTE_COUNT_COLUMN = 'credit_note_counter';
+    const CREDIT_NOTE_PREFIX = 'CDE';
```

Comment:
If there is a const for credit note prefix, there should be a rma prefix also :-) But it is hard coded (look above)